config/spryk/spryks/Spryker/Zed/Business/AddZedBusinessFactoryMethod.yml

```diff
@@ -38,3 +38,60 @@ arguments:
 preSpryks:
     - AddZedBusinessFactory
 
+postSpryks:
```

Comment:
These has to go to the `AddCrudFacade` Spryk. Do not change existing spryks like this. `AddZedBusinessFactoryMethod` is a generic Spryk that can be used by others. Everyone who just wants to add a speific method to a factory will now get all the methods you define here which is not correct.

Move all `postSpryks` added *here* to the `postSpryk` definition of the `AddCrudFacade` Spryk.